### PR TITLE
Fix: Flutter platform wizard persistence

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

Fixes a bug in which the Add Flutter platform wizard doesn't remember platform choice

## Test Plan

Manual

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/console/+layout.svelte [hunk 1]
```diff
@@ -50,14 +51,10 @@
...
         !$page.url.pathname.includes('/console/account') &&
         !$page.url.pathname.includes('/console/onboarding')}>
-    <svelte:fragment slot="header">
-        <Header />
-    </svelte:fragment>
-    <svelte:fragment slot="side">
-        <SideNavigation />
-    </svelte:fragment>
+    <Header slot="header" />
+    <SideNavigation slot="side" />
```
Comment (TorstenDittmann) [inline src/routes/console/+layout.svelte:55]: Didn't know this works 👀

File: src/lib/layout/footer.svelte [hunk 2]
```diff
@@ -1,47 +1,53 @@
-<footer class="main-footer">
-    <div class="container">
-        <ul class="inline-links is-center">
+<script>
```
Comment (TorstenDittmann) [inline src/lib/layout/footer.svelte:1]: I think you got some footer from another PR in here 😉

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

File: src/lib/layout/footer.svelte (hunk 2 above)
Comment (TGlide) [reply src/lib/layout/footer.svelte]: Oops! Should be fixed now

File: src/routes/console/project-[project]/overview/platforms/wizard/flutter/step1.svelte [hunk 3]
```diff
@@ -23,7 +23,13 @@
...
     }
 
-    let platform: Platform = Platform.Android;
+    function isPlatform(value: string): value is Platform {
+        return Object.values(Platform).includes(value as Platform);
+    }
+
+    let platform: Platform = isPlatform($createPlatform.type)
+        ? $createPlatform.type
+        : Platform.Android;
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/overview/platforms/wizard/flutter/step1.svelte:32]: That does only seem to persist once 👍🏻 

https://user-images.githubusercontent.com/1759475/218757230-9e8b2116-caed-4b98-811f-5583e9ed0c9e.mov

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**